src/frontend/js/components/SolutionData.js

```diff
@@ -264,11 +264,11 @@ class SolutionData extends React.Component {
             <Chip className={classes.icon} label={'\u266D'} onClick={() => this.handleModification('b')} />
             <Chip className={classes.icon} label={'\u266F'} onClick={() => this.handleModification('#')} />
             <Chip className={classes.icon} label={'\u266E'} onClick={() => this.handleModification('n')} />
-            <Chip className={classes.icon} label={<img className={classes.whole} src='../../../../assets/images/whole.png' />} onClick={() => this.modifyDuration(1)} />
-            <Chip className={classes.icon} label={<img className={classes.image} src='../../../../assets/images/half.png' />} onClick={() => this.modifyDuration(2)} />
+            <Chip className={classes.icon} label={<img className={classes.whole} src='/assets/images/whole.png' />} onClick={() => this.modifyDuration(1)} />
```

Comment:
accidental inclusion?

Reply:
No, I changed the src path. it's needed for copying the assets folder when building